### PR TITLE
Address SI notion

### PR DESCRIPTION
**Pull request by aMegaSloth**
#### Summary
Interface "Normalize usage of SI units and include spaces in JSON for volume entries"

#### Purpose of change
Brings in line the SI units based on SI unit usage rules. SI unit rules outlined are [here](https://physics.nist.gov/cuu/Units/checklist.html). Specifically
` #15. Unit spacing. There is a space between the numerical value and unit symbol, even when the value is used in an adjectival sense, except in the case of superscript units for plane angle.`

#### Describe the solution
Go into json files and add spaces.

#### Testing
Did a fresh Build after making json changes. Checked a bunch of items and they seem fine. Didn't expect much really.

### CODE REVIEW

**Comment by Maleclypse** [comment]
@Faalagorn

**Comment by Hirmuolio** [comment]
While at it the ml/mL could be unified to be consistent. Both are valid ways to write it but it would be nicer to only use one.

**Comment by aMegaSloth** [comment]
Originally I was going though the code and noticed a few JSON entries
`"volume": "20ml"`
All of those were changed to:
`"volume": "20 ml"`

Usage of ml -> mL was on the display side of things like string descriptors, and ect.
Space between the number and unit were added to a punch of string name, but unchanged in code, so for example:

`steel drum (200L)` -> `steel drum (200 L)`, but in the code it remains `tank_55gal_drum` in the json id.

**Comment by Night-Pryanik** [comment]
There are also TONS of strings like `5.56x45mm`.

**Comment by aMegaSloth** [comment]
> There are also TONS of strings like `5.56x45mm`.

I thought about doing the ammo as well but wasn't sure if that was correct since I don't know guns. If that is something that could be/needs to change, I can go ahead and do that.

**Comment by Faalagorn** [comment]
> > There are also TONS of strings like `5.56x45mm`.
> 
> I thought about doing the ammo as well but wasn't sure if that was correct since I don't know guns. If that is something that could be/needs to change, I can go ahead and do that.

I'm living in Europe, and in Poland we do have spaces between mm in ammo names, not sure about other countries, but from what I've seen and understand, the thing with guns is that some use imperial units in United states where there's no unit given at all .338, .45 ACP etc., while the ones with mm in it are typically used without a space. I've seen every game so far not using spaces, so let's leave them like that for now I guess, but I'm open to ideas.

Glad the spaces are being unified, as it was pointed in one of my PRs. 

You may go ahead and add it to Manual of style too, so hopefully more people like me will be aware of these rules: https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md

File: data/json/items/ammo.json [hunk 1]
```diff
@@ -380,7 +380,7 @@
     "color": "dark_gray",
     "description": "Plastic jacketed copper cable of the type used in small electronics.",
     "material": [ "plastic", "copper" ],
-    "//": "Calculated off of copper wire with an extra 0.5mm of diameter and 0.66g.",
+    "//": "Calculated off of copper wire with an extra 0.5 mm of diameter and 0.66g.",
```
Comment (Faalagorn) [inline data/json/items/ammo.json:383]: 0.66 g?

Comment (aMegaSloth) [reply data/json/items/ammo.json]: Thanks for catching that. I also updated it to be 600 mg for milligrams as opposed to fraction of grams.

**Comment by I-am-Erk** [comment]
Unfortunately with the merge conflicts I am moving this to abandoned but desired PRs. It should probably be done in smaller PRs to prevent this kind of giant conflict.

**Pull request closed without merging**